### PR TITLE
44 header changes

### PR DESCRIPTION
**Pull request by mohitgusain**
#44 
Submenu styling for header

### CODE REVIEW

File: public/wp-content/themes/glific/src/scss/header/style.scss [hunk 1]
```diff
@@ -66,3 +66,74 @@
 		}
 	}
 }
+
+.desktop-menu {
+	li{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/header/style.scss]:
```suggestion
	li {
```

File: public/wp-content/themes/glific/src/scss/header/style.scss [hunk 2]
```diff
@@ -66,3 +66,74 @@
...
+	}
+	li a {
+		@extend .font-heebo-regular, .fz-md-16, .leading-md-20, .text-theme-dark-green, .text-decoration-none, .w-full;
+	}
+	.sub-menu {
+		@extend .bg-white, .position-absolute, .px-md-3, .mt-28, .w-220, .py-3, .left-n12\.5, .rounded-11;
+		display: none;
+		li {
+			@extend .border-bottom, .border-theme-pewter;
+			a{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/header/style.scss]:
```suggestion
			a {
```

File: public/wp-content/themes/glific/src/scss/header/style.scss [hunk 3]
```diff
@@ -66,3 +66,74 @@
...
+				@extend .w-full, .py-4, .pl-11, .d-block, .fz-14;
+				:hover {
+					@extend .text-theme-dark-green, .bg-theme-pewter;
+				}
+			}
+			a:hover{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/header/style.scss]:
```suggestion
			a:hover {
```

File: public/wp-content/themes/glific/src/scss/header/style.scss [hunk 4]
```diff
@@ -66,3 +66,74 @@
...
+			margin-left: 5px;
+		}
+	}
+}
+.sub-menu {
+	@extend .list-unstyled, .pt-md-2;
+}
+
+.mobile-primary-menu {
+	li{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/header/style.scss]:
```suggestion
	li {
```

File: public/wp-content/themes/glific/src/scss/header/style.scss [hunk 5]
```diff
@@ -66,3 +66,74 @@
...
+		@extend .position-relative;
+	}
+	.sub-menu {
+		@extend .bg-theme-secondary, .position-absolute, .px-md-3, .w-220, .py-3, .rounded-11, .right-0;
+		display: none;
+		li {
+			@extend .border-bottom, .border-theme-pewter, .mb-0, .px-2;
+			a{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/header/style.scss]: @mohitgusain Can you please use any SCSS formatted? There are lots of formatted available in the market place.

```suggestion
			a {
```

File: public/wp-content/themes/glific/src/scss/style.scss [hunk 6]
```diff
@@ -85,6 +85,11 @@ body {
 .team-bg-image {
 	background-image: url('dist/images/bg-team.svg') !important;
 }
+
+.left-10p{
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/style.scss]: Formate

File: public/wp-content/themes/glific/src/js/main.js [hunk 7]
```diff
@@ -26,9 +26,27 @@ jQuery(document).ready(function() {
 		jQuery(target).addClass('d-flex').removeClass('d-none');
 	});
 
-	jQuery('.show-more-blogs').on('click', show_more_blogs);
+	jQuery('.desktop-menu .menu-item-has-children').hover(function() {
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/js/main.js:29]: @mohitgusain Could we have done this with CSS? Any thoughts on using these two different approaches?

**Review by Satendra-SR — COMMENTED** [review]
@mohitgusain Added a few comments, have a look. 
Also, you are daily interacting with SCSS. It would be good to have a formatter.

File: public/wp-content/themes/glific/src/js/main.js (hunk 7 above)
Comment (mohitgusain) [reply public/wp-content/themes/glific/src/js/main.js:29]: No, this is not possible with CSS for now. We are using the other classes on it.

Comment (Satendra-SR) [reply public/wp-content/themes/glific/src/js/main.js:29]: @mohitgusain any update on this?

Comment (Satendra-SR) [reply public/wp-content/themes/glific/src/js/main.js:29]: Ok.

**Review by Satendra-SR — APPROVED** [review]
Looks good. Please use an SCSS formater now onwards.

**Pull request merged**